packages/api/src/controllers/access-control.ts

```diff
@@ -250,6 +271,30 @@ app.post(
   }
 );
 
+app.post("/suspend/:key", authorizer({ anyAdmin: true }), async (req, res) => {
```

Comment:
Can we add a get handler to check whether something is suspended and/or a handler for listing all entries

Reply:
Added!